CHANGELOG.md

```diff
@@ -10,6 +10,27 @@ to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ### Breaking changes
 
+### Deprecations
+
+### New features
+
+### Fixed bugs
+
+## [0.33.0] - 2025-09-03
+
+### Release highlights
+
+* `jj undo` is now *sequential*: invoking it multiple times in sequence repeatedly undoes
+  actions in the operation log. Previously, `jj undo` would only undo *the most recent*
+  operation in the operation log, a "fixed point". As a result, a new `jj redo` command
+  has been added.
+
+* Experimental support for improving query performance over filesets and file
+  queries (like `jj log path/to/file.txt`) has been added. To enable this, you
+  must use the `jj debug index-changed-paths` command.
```

Comment:
The way I first read this made me think that `jj debug index-changed-paths` is used as a replacement for some other command. I think it's the "you must use the ... command" bit that confused me. Maybe replace the sentence by "Run ... to create the index." is clearer?

Reply:
Good point.